### PR TITLE
Added additional information about user roles managing

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

_Added additional information about user roles managing. Editing/Deleting Roles._

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/system/permissions-user-roles.html/>._

### CODE REVIEW

File: src/system/permissions-user-roles.md [hunk 1]
```diff
@@ -74,3 +74,25 @@ To give someone restricted access to the Admin, the first step is to create a ro
...
     ![User accounts assigned to the role]({% link images/images/permissions-role-users.png %}){: .zoom}
     _Role Users_
+
+## Edit a role
+
+1. On the _Admin_ sidebar, go to **System** > _Permissions_ > **User Roles**.
+
+1. Locate the role using filters above the grid and click the role name.
+
+1. Make needed changes.
```
Comment (leslietilling) [inline src/system/permissions-user-roles.md:84]:
```suggestion
1. Make needed changes.
```
```suggestion
1. Make needed changes.

   Review the steps for creating a user role for information. about the role settings.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a minor addition to clarify "changes".

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**